tests/SprykerSdkTest/Spryk/Console/SprykRunSpryksTest.php

```diff
@@ -46,4 +46,46 @@ public function testExecutesSpryks(): void
         $this->assertDirectoryExists($this->tester->getVirtualDirectory() . 'vendor/spryker/spryker/Bundles/FooBar/firstDirectory');
         $this->assertDirectoryExists($this->tester->getVirtualDirectory() . 'vendor/spryker/spryker/Bundles/FooBar/secondDirectory');
     }
+
+    /**
+     * @return void
+     */
+    public function testExecutesNotUniqueSpryks(): void
+    {
+        /** @var \SprykerSdk\Spryk\Console\SprykRunConsole $command */
+        $command = $this->tester->getClass(SprykRunConsole::class);
+        $tester = $this->tester->getConsoleTester($command);
+
+        $arguments = [
+            'command' => $command->getName(),
+            SprykRunConsole::ARGUMENT_SPRYK => 'SprykWithNotUniqueSpryks',
+            '--mode' => 'core',
+        ];
+
+        $tester->execute($arguments);
+
+        $this->assertDirectoryExists($this->tester->getVirtualDirectory() . 'vendor/spryker/spryker/Bundles/FooBar/firstDirectory');
+        $this->assertDirectoryExists($this->tester->getVirtualDirectory() . 'vendor/spryker/spryker/Bundles/FooBar/secondDirectory');
+    }
+
+    /**
+     * @return void
+     */
+    public function testExecutesSpryksIfSprykeHasExecutedSpryks(): void
```

Comment:
Same as for the other method. Always think that in half a year everyone who reads the test should be able to understand them without the need to debug. 

You also have a typo (Spryke) and I think you wanted to use `Excluded` and not `Executed`.